src/api/client/cache.ts

```diff
@@ -49,13 +49,15 @@ const getChannelKeyArgs = (args: Record<string, GetChannelsConnectionQueryVariab
 const getSearchKeyArgs = (args: Record<string, SearchQueryVariables['whereVideo']> | null) => {
   const text = args?.text || ''
   const licenseIdIn = args?.whereVideo?.licenseId_in || []
-  const hasMarketingEq = args?.whereVideo?.hasMarketing_eq || ''
-  const isExplicitEq = args?.whereVideo?.isExplicit_eq || ''
+  const hasMarketingEq = args?.whereVideo?.hasMarketing_eq || true
+  const isExplicitEq = args?.whereVideo?.isExplicit_eq || true
```

Comment:
Why are we assuming `true` for those if they're missing? This can lead to wrong results